web/sites/default/config/cloudy.settings.yml

```diff
@@ -1,22 +1,3 @@
-bootstrap_barrio_library: development
-bootstrap_barrio_sidebar_position: both
-bootstrap_barrio_content_offset: '0'
-bootstrap_barrio_sidebar_first_width: '3'
-bootstrap_barrio_sidebar_first_offset: '0'
-bootstrap_barrio_sidebar_second_width: '4'
-bootstrap_barrio_sidebar_second_offset: '0'
-bootstrap_barrio_fluid_container: 0
-bootstrap_barrio_button_size: ''
-bootstrap_barrio_button_outline: 0
-bootstrap_barrio_navbar_toggle: navbar-toggleable-lg
-bootstrap_barrio_navbar_container: 1
-bootstrap_barrio_navbar_top_position: ''
-bootstrap_barrio_navbar_top_color: navbar-dark
-bootstrap_barrio_navbar_top_background: ''
-bootstrap_barrio_navbar_position: ''
-bootstrap_barrio_navbar_color: navbar-dark
-bootstrap_barrio_navbar_background: ''
```

Comment:
These were safe to delete—leftovers from removing Barrio ages ago. I had not seen this config in previous PRs because we've only ever updated this with logo changes. Those changes were small enough I had never even seen this config before. Good opportunity to clean it up manually.